7.1 Snapshot.md

```diff
@@ -0,0 +1,52 @@
+# 7.1 Snapshot
+
+## 7.1.1 Introduction
+This section defines the *Snapshot* protocol.
+
+
+## 7.1.2 Dependencies
+The *Snapshotting* protocol depends on:
+
++ Timestamps: Core module to enable pruning.
++ Tip Selection Algorithm: No messages can be referenced for the part of the Tangle that is being pruned.
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
```

Comment:
OTV+FPCS must decide on whether a transaction is liked or disliked much before the point of pruning. See line 16

I assume similar to FPC also FPCS has a maximum round after which the protocol force stops (maxRound). the maximum round should not be reached, because otherwise there is possibility for agreement failures. 

Lets talk quickly FPC: there the typical round number when conflicts even with metastabillity attacks were resolved in FPC was 10 (so 10*10sec). And the FPC protocol kind of ensures this. We should also require this property for FPCS and my feeling is that it will perform actually better.